UI/Components/DetailedTimer.cs

```diff
@@ -347,7 +350,22 @@ public void Update(IInvalidator invalidator, LiveSplitState state, float width,
                         BestSegmentTime.Text = TimeFormatConstants.DASH;
                 }
                 if (state.CurrentSplitIndex >= 0)
-                    SplitName.Text = state.Run[state.CurrentSplitIndex + lastSplitOffset].Name;
+                {
+                    string name = state.Run[state.CurrentSplitIndex + lastSplitOffset].Name;
+
+                    bool isSubsplit = name.StartsWith("-") && state.CurrentSplitIndex + lastSplitOffset < state.Run.Count;
```

Comment:
This should be `< state.Run.Count - 1`.  The last split shouldn't ever show up as a subsplit.

Reply:
Oh yes of course, fixed in new commit